controllers/common/common_status_update.go

```diff
@@ -185,19 +188,19 @@ func GetClusterDecisions(
 			continue
 		}
 
-		plcDecisions, plcPlacements, err := GetPolicyPlacementDecisions(ctx, c, rootPolicy, &pbList.Items[i])
+		plcDecisions, plcPlacements, err := GetPolicyRolloutClusters(ctx, c, rootPolicy, &pbList.Items[i])
 		if err != nil {
 			return nil, nil, err
 		}
 
-		if len(plcDecisions) == 0 {
+		if len(plcDecisions.ClustersToRollout) == 0 {
 			log.Info("No placement decisions to process for this policy from this non-restricted binding",
```

Comment:
Change this log too? something like` No rollout`...

Reply:
It becomes weird here because the PlacementBinding has its own configuration that can filter the placement. I've updated it to try to be a little clearer--let me know if it makes sense.